packages/contracts-bedrock/test/DelayedVetoable.t.sol

```diff
@@ -164,15 +164,66 @@ contract DelayedVetoable_HandleCall_TestFail is DelayedVetoable_Init {
         success2;
     }
 
+    /// @dev The delay is inititially set to zero and the call is immediately forwarded.
+    function testFuzz_handleCall_initialForwardingImmediately_succeeds(
+        bytes calldata inData,
+        bytes calldata outData
+    )
+        external
+    {
+        _assumeNonzeroData(inData);
+        _assumeNoClash(inData);
+
+        // Reset the delay to zero
+        vm.store(address(delayedVetoable), bytes32(uint256(0)), bytes32(uint256(0)));
+
+        vm.mockCall(target, inData, outData);
+        vm.expectEmit(address(delayedVetoable));
+        vm.expectCall({ callee: target, data: inData });
```

Comment:
was confused for a second because i normally expect `expectEmit` to be directly followed by an `emit`

Reply:
Good catch. Fixed.